### PR TITLE
Dockerfile for xs-cli

### PR DESCRIPTION
**Pull request by radsoulbeard**

### CODE REVIEW

File: xs-cli/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# ps needs to be available to be able to be used in docker.inside, see https://issues.jenkins-ci.org/browse/JENKINS-40101
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends procps unzip && \
+    rm -rf /var/lib/apt/lists/*
+
+# add group & user
+RUN addgroup -gid 1000 piper && \
+    useradd piper --uid 1000 --gid 1000 --shell /bin/bash --create-home && \
+    mkdir -p ${XSCLI} && \
+    cd ${XSCLI} && \
```
Comment (CCFenner) [inline xs-cli/Dockerfile]: could be removed if `unzip` is used with `-d ${XSCLI}` option

File: xs-cli/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# XS CLI Docker File
+## Description
+To bring an XS application to a HANA XS Server you need to deploy the application with the xs command line tool. This Dockerfile can wrap the xs command line client and the resulting image is intended to run with our Jenkins pipeline library [project "Piper"][piper].
```
Comment (1084565) [inline xs-cli/README.md]:
```suggestion
To bring an SAP HANA XS Advanced application to a SAP HANA Server you need to deploy the application with the xs command line tool. This Dockerfile can wrap the xs command line client and the resulting image is intended to run with the Jenkins pipeline library [project "Piper"][piper]. 
```

File: xs-cli/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# XS CLI Docker File
+## Description
+To bring an XS application to a HANA XS Server you need to deploy the application with the xs command line tool. This Dockerfile can wrap the xs command line client and the resulting image is intended to run with our Jenkins pipeline library [project "Piper"][piper]. 
+
+## Requirements
+* General requirements can be found in the [repository readme][general]
+* An S-User for [SAP ONE][sapone]
+* Download XS command line client ```XS_CLIENT00P_<version>.ZIP``` for Linux on x86_64 from [SAP ONE][sapone]
```
Comment (1084565) [inline xs-cli/README.md]: Please use XS consistently. I think it's used as "XS Command-Line Interface" or XS CLI. @SarahNoack can you please check the naming in this doc?

File: xs-cli/README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+Example:
+```
+docker build -t ppiper/xs-cli --build-arg XSZIP=XS_CLIENT00P_<version>.ZIP --file Dockerfile https://github.com/SAP/devops-docker-images.git#:xs-cli
+```
+
+## How to execute it
+Assuming you have built the image with using the tag `ppiper/xs-cli` you can run it with:
+
+```
+docker run  ppiper/xs-xli xs <command>
```
Comment (1084565) [inline xs-cli/README.md]:
```suggestion
docker run  ppiper/xs-cli xs <command>
```

File: xs-cli/README.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
+# XS CLI Docker File
+## Description
+To bring an SAP HANA XS Advanced application to a SAP HANA Server you need to deploy the application with the xs command line tool. This Dockerfile can wrap the xs command line client and the resulting image is intended to run with the Jenkins pipeline library [project "Piper"][piper].
```
Comment (SarahNoack) [inline xs-cli/README.md]:
```suggestion
To bring an SAP HANA XS Advanced application to an SAP HANA Server, deploy the application with the XS command-line tool. With this Dockerfile, you can wrap the XS command-line client and run the resulting image with the Jenkins pipeline library of [project "Piper"][piper]. 
```

File: xs-cli/README.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
+# XS CLI Docker File
+## Description
+To bring an SAP HANA XS Advanced application to a SAP HANA Server you need to deploy the application with the xs command line tool. This Dockerfile can wrap the xs command line client and the resulting image is intended to run with the Jenkins pipeline library [project "Piper"][piper]. 
+
+## Requirements
+* General requirements can be found in the [repository readme][general]
+* An S-User for [SAP ONE][sapone]
+* Download XS command line client ```XS_CLIENT00P_<version>.ZIP``` for Linux on x86_64 from [SAP ONE][sapone]
```
Comment (SarahNoack) [inline xs-cli/README.md]:
```suggestion
* Download XS command-line client ```XS_CLIENT00P_<version>.ZIP``` for Linux on x86_64 from [SAP ONE][sapone]
```

File: xs-cli/README.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
+# XS CLI Docker File
+## Description
+To bring an SAP HANA XS Advanced application to a SAP HANA Server you need to deploy the application with the xs command line tool. This Dockerfile can wrap the xs command line client and the resulting image is intended to run with the Jenkins pipeline library [project "Piper"][piper]. 
+
+## Requirements
+* General requirements can be found in the [repository readme][general]
+* An S-User for [SAP ONE][sapone]
+* Download XS command line client ```XS_CLIENT00P_<version>.ZIP``` for Linux on x86_64 from [SAP ONE][sapone]
+
+## How to build it
```
Comment (SarahNoack) [inline xs-cli/README.md]:
```suggestion
## How to Build It
```

File: xs-cli/README.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+To bring an SAP HANA XS Advanced application to a SAP HANA Server you need to deploy the application with the xs command line tool. This Dockerfile can wrap the xs command line client and the resulting image is intended to run with the Jenkins pipeline library [project "Piper"][piper]. 
+
+## Requirements
+* General requirements can be found in the [repository readme][general]
+* An S-User for [SAP ONE][sapone]
+* Download XS command line client ```XS_CLIENT00P_<version>.ZIP``` for Linux on x86_64 from [SAP ONE][sapone]
+
+## How to build it
+
+This image will not be provided on hub.docker.com. You need to [build][dockerbuild] this Dockerfile locally before using it. Here you can find a [tutorial][xsclient] how to get the xs command line client package.
```
Comment (SarahNoack) [inline xs-cli/README.md]:
```suggestion
This image is not provided on hub.docker.com. Instead, [build][dockerbuild] this Dockerfile locally before using it. Here, you can find a [tutorial][xsclient] on how to get the XS command-line client package.
```

File: xs-cli/README.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+
+### Build Arguments
+| Argument | Description |
+| ---------| ------------|
+| **XSZIP** | Path to your XS CLI zip file |
```
Comment (SarahNoack) [inline xs-cli/README.md]:
```suggestion
| **XSZIP** | Path to your XS CLI ZIP file |
```

File: xs-cli/README.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+
+Example:
+```
+docker build -t ppiper/xs-cli --build-arg XSZIP=XS_CLIENT00P_<version>.ZIP --file Dockerfile https://github.com/SAP/devops-docker-images.git#:xs-cli
+```
+
+## How to execute it
```
Comment (SarahNoack) [inline xs-cli/README.md]:
```suggestion
## How to Execute It
```

File: xs-cli/README.md [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+Assuming you have built the image with using the tag `ppiper/xs-cli` you can run it with:
```
Comment (SarahNoack) [inline xs-cli/README.md]:
```suggestion
Assuming you have built the image by using the tag `ppiper/xs-cli`, you can run it with:
```

**Pull request merged**